gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-list-row/api-runtime-logs-list-row.component.html

```diff
@@ -0,0 +1,68 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="log-row" [class.first]="index === 0">
+  <div class="log-row__left">
+    <div class="log-row__line-1">
+      <div>{{ log.timestamp }}</div>
+      <span class="gio-badge-neutral">Request ended</span>
+    </div>
+
+    <div class="log-row__line-2">
+      <div class="log-row__line-2__request">
+        <div class="log-row__line-2__data">
+          <div class="property">Entrypoint:</div>
```

Comment:
As for the moment we don't have the entrypoint I would suggest to reword it to the following.

```suggestion
          <div class="property">Http method:</div>
```